experimental/docker-stacks-and-bundles.md

```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy vossibility-stack  vossibility
```

Comment:
Correct. Could you update this document to show how it should be used (with the `--bundle-file` flag?)